h/static/scripts/components/test/AnnotationCard-test.js

```diff
@@ -68,21 +68,21 @@ describe('AnnotationCard', () => {
       '../hooks/use-update-moderation-status': {
         useUpdateModerationStatus: fakeUseUpdateModerationStatus,
       },
-      '../utils/api': {
+      '../util/api': {
```

Comment:
The `h/static/scripts/` directory already contained a `util` subdirectory. In the client we use singular `util`, in the LMS we use plural `utils`. I kept the existing directory name.